Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -92,6 +101,21 @@ bool fetchInstructionsMode::ReceiveTeleopInstructions(attitudeManager* attitudeM
     return true;
 }
 
+bool fetchInstructionsMode:: isArmed()
+{
+    //Range of PPM percentage to be considered ARMED is 95-100%
+    const uint8_t MIN_ARM_VALUE = 50;
+    const uint8_t MAX_ARM_VALUE = 100;
+    if (_TeleopInstructions.PPMValues[ARM_DISARM_CHANNEL_INDEX] >= MIN_ARM_VALUE && _TeleopInstructions.PPMValues[ARM_DISARM_CHANNEL_INDEX] <= MAX_ARM_VALUE)
```

Comment:
Also, is checking that the value is less than the max arm value necessary? It shouldn't be necessary for a 2-state switch right?